driver.py

```diff
@@ -0,0 +1,245 @@
+import getopt
+import subprocess
+import sys
+
+from util import *
+
+def help():
+    """ Print help string """
+
+    help_str = """
+python3 driver.py 
+
+Description
+-----------
+    -i, --install 
+        install all dependencies from the feature set
+    
+    -c, --check
+        run `cargo check` 
+
+    -b, --build
+        run `cargo build` and build all dependencies from the feature set 
+
+    -t, --test 
+        run `cargo test` and test all dependencies from the feature set
+    
+    -f, --format
+        run `cargo fmt --all`
+        
+    -l, --lint 
+        run `cargo clippy`
+        
+    -C, --clean 
+        remove all generated files
+    
+    -A, --all_features
+        set all features on 
+
+    -L, --list_features
+        print active feaures
+
+    -a, --add_feature=<aby,c_front>
```

Comment:
I'd recommend a 'set' interface (that sets all) rather than an 'add'/'remove' interface. That may be personal opinion?